create_aws_application

```diff
@@ -0,0 +1,71 @@
+#!/bin/sh
+# this is the name of the application you want to create
+NAME=$1
+# these are all the environments. the buckets will be named NAME-ENVIRONMENT
+ENVIRONMENTS=(master develop testing)
+# region where you want to create the bucket
+REGION=eu-central-1
+# aws-cli profile to use (default default)
+PROFILE=${2:-default}
```

Comment:
Have you checked that these variable names don't clash with something else? In particular `$PROFILE` looks like a very common name...